src/executor/ms_executor.c

```diff
@@ -64,11 +79,16 @@ void	ft_execute(t_ms *ms, t_cmd_list *curr)
 
 	cmdpath = ft_search_cmd(ms, curr);
 	if (cmdpath == NULL)
+	{
 		err = 127;
+		ft_prnt_stderr(NULL, curr->cmd->words->word, 2);
+	}
 	else
 	{
 		envp = ft_create_envp(ms);
-		err = execve(cmdpath, curr->cmd->argv, envp);
+		if (execve(cmdpath, curr->cmd->argv, envp) == -1)
+			err = 127;
+		ft_prnt_stderr("minishell: ", curr->cmd->words->word, errno);
```

Comment:
print "minishell" without ':'
as in `ft_prnt_stderr("minishell", ...);`

current output without changes:
```
ms/kjzl@fedora-kajus:~/Desktop/42/minishell$ ./a8
[DBG] parsed AST: ./a8
minishell: : ./a8: No such file or directoryms/kjzl@fedora-kajus:~/Desktop/42/minishell$
```